channels/google/accounts/managing-accounts.delete-note.rst

```diff
@@ -0,0 +1,6 @@
+.. note:: 🧹 **Additional Steps Required**
+
+   Groups need an assigned an address to send outgoing emails.
+   If you delete a group’s assigned address,
+   agents belonging to that group won’t be able to send emails
+   until you assign it a new one.
```

Comment:
I removed the following note from Google's Managing Accounts page:

```restructuredtext
.. note:: ⚠ **No Additional Steps Required**

Unlike the Email channel, Zammad removes your configured email addresses 
together with your account if you delete it. This is, because we can reliabely 
determine them as gmail addresses which you can't use on other servers at that moment.
```

I thought about it and decided it was okay to leave undocumented: even if it works contrary to the user's expectations, there's no way it can cause any serious problems.

Reply:
I'd prefer to keep this part inside the documentation, because it took me 30-45 minutes during documentation to realise that this is working like it's supposed to be.

I'm very sure that this will, at some point, other wise cause a bug report which then relies on me remembering that this is working as intended. That's potentially dangerous.